core/whistle_couch-1.0.0/src/couch_util.erl

```diff
@@ -132,7 +131,7 @@ db_classification(?KZ_ACDC_DB) -> 'aggregate';
 db_classification(?WH_SERVICES_DB) -> 'aggregate';
 db_classification(?KZ_PORT_REQUESTS_DB) -> 'aggregate';
 db_classification(?KZ_WEBHOOKS_DB) -> 'aggregate';
-db_classification(<<?WNM_DB_PREFIX_L, _Prefix:5/binary>>) -> 'numbers';
+db_classification(<<"numbers/", _Prefix:5/binary>>) -> 'numbers';
```

Comment:
Simplifying what got included here. We were hard-coding the other two variants, seemed silly to have a `include_lib` for this one variation.